### PR TITLE
Sending in the Clowns: More Clown outfits.

### PR DESCRIPTION
**Pull request by theselfish**
## About The Pull Request

image
image
Credits are in the files.

## Why It's Good For The Game

I like Clowns, a lot of users like clowns.
Other than the weird fact that there's orange/red/blue/etc uniforms but no matching masks, the other two outfits allow variety.

Or something.
I also re-added clown cryto as...We aren't Skyrat, and they removed it during the Clown Purge. Barely anyone does cryto. If this ends up being trouble, it can be reverted.

Oh and thank you to https://github.com/Bubberstation/Bubberstation/pull/848 for the vending code.

## Proof Of Testing

See the screenshots.

## Changelog

:cl:
add: Added various clown clothing into the autodrobe.
add: Clown Cryto is back.
code: Modular Vending code from Skyrat, but for us.
/:cl:

### CODE REVIEW

File: modular_zubbers/modules/vending/vending.dm [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+/obj/machinery/vending
+	/// Additions to the `products` list  of the vending machine, modularly. Will become null after Initialize, to free up memory.
+	var/list/zubbers_products
+	/// Additions to the `product_categories` list of the vending machine, modularly. Will become null after Initialize, to free up memory.
+	var/list/zubbers_product_categories
+	/// Additions to the `premium` list  of the vending machine, modularly. Will become null after Initialize, to free up memory.
+	var/list/zubbers_premium
+	/// Additions to the `contraband` list  of the vending machine, modularly. Will become null after Initialize, to free up memory.
+	var/list/zubbers_contraband
```
Comment (Majkl-J) [inline modular_zubbers/modules/vending/vending.dm:11]: I could've sworn we already had this somewhere. Guess not. alas, add a whitespace
```suggestion
/obj/machinery/vending
	/// Additions to the `products` list  of the vending machine, modularly. Will become null after Initialize, to free up memory.
	var/list/zubbers_products
	/// Additions to the `product_categories` list of the vending machine, modularly. Will become null after Initialize, to free up memory.
	var/list/zubbers_product_categories
	/// Additions to the `premium` list  of the vending machine, modularly. Will become null after Initialize, to free up memory.
	var/list/zubbers_premium
	/// Additions to the `contraband` list  of the vending machine, modularly. Will become null after Initialize, to free up memory.
	var/list/zubbers_contraband

```

**Review by Majkl-J — CHANGES_REQUESTED** [review]
You need to comment that you edited skyrat uncomments (As cursed as that may be), otherwise all good

File: code/modules/clothing/masks/gasmask.dm [hunk 2]
```diff
@@ -296,7 +296,11 @@ GLOBAL_LIST_INIT(clown_mask_options, list(
 		"The Madman" = image(icon = src.icon, icon_state = "joker"),
 		"The Rainbow Color" = image(icon = src.icon, icon_state = "rainbow")
 		)
-	//AddElement(/datum/element/swabable, CELL_LINE_TABLE_CLOWN, CELL_VIRUS_TABLE_GENERIC, rand(2,3), 0) //SKYRAT EDIT REMOVAL
+	AddElement(/datum/element/swabable, CELL_LINE_TABLE_CLOWN, CELL_VIRUS_TABLE_GENERIC, rand(2,3), 0)
+
+// BUBBERSTATION EDUT
```
Comment (theselfish) [inline code/modules/clothing/masks/gasmask.dm:301]:
```suggestion
// BUBBERSTATION EDIT
```

File: code/modules/clothing/under/jobs/civilian/clown_mime.dm [hunk 3]
```diff
@@ -32,14 +32,13 @@
...
-//SKYRAT EDIT REMOVAL BEGIN
-/*
 /obj/item/clothing/under/rank/civilian/clown/Initialize(mapload)
 	. = ..()
 	AddComponent(/datum/component/squeak, list('sound/items/bikehorn.ogg'=1), 50, falloff_exponent = 20) //die off quick please
 	AddElement(/datum/element/swabable, CELL_LINE_TABLE_CLOWN, CELL_VIRUS_TABLE_GENERIC, rand(2,3), 0)
-*/
-//SKYRAT EDIT REMOVAL END
+
+// BUBBERSTATION EDUT
```
Comment (theselfish) [inline code/modules/clothing/under/jobs/civilian/clown_mime.dm:40]:
```suggestion
// BUBBERSTATION EDIT
```

**Comment by plsleavemealon** [comment]
Is there any chance to be able to add these things to the loadout as a generalist option?

**Comment by theselfish** [comment]
> Is there any chance to be able to add these things to the loadout as a generalist option?

Just go to an autodrobe.

**Comment by plsleavemealon** [comment]
I can't customize the physical description and the name of the item if I am forced to buy it from an autodrobe.

**Comment by theselfish** [comment]
> I can't customize the physical description and the name of the item if I am forced to buy it from an autodrobe.

Fine.

**Comment by plsleavemealon** [comment]
Thank you!

**Review by Majkl-J — APPROVED** [review]
Honk (Also I really hope the clown cytology won't become a mess)

**Comment by plsleavemealon** [comment]
Aww the pink clown skirt is role locked :(

**Pull request merged**